provider/resources.go

```diff
@@ -2469,6 +2469,36 @@ func ProviderFromMeta(metaInfo *tfbridge.MetadataInfo) *tfbridge.ProviderInfo {
 						AltTypes:  []tokens.Type{awsType(iamMod, "documents", "PolicyDocument")},
 						Transform: tfbridge.TransformJSONDocument,
 					},
+					"inline_policy": {
+						// inline_policy is an array of policy objects. It is allowed to provided an empty list
```

Comment:
Grammar: "allowed to be provided as an empty list" or something like this, sounds odd as written.